chromium/utils/translate-en-messages.py

```diff
@@ -1,139 +1,119 @@
-            translated_msgs[key] = { 'message': translated_msg }
-            continue
-        if key not in messages:
-            original_msg = translated_msg = en_messages[key]['message']
+            translated_msgs[key] = value
+        else:
             try:
-                translator = Translator(provider=provider if provider else '', to_lang=lang_code)
-                translated_msg = translator.translate(original_msg).replace('&quot;', "'").replace('&#39;', "'")
-                if any(flag in translated_msg for flag in fail_flags):
-                    translated_msg = original_msg
-            except Exception as e:
-                print_trunc(f'Translation failed for key "{key}" in {lang_code}/messages.json: {e}')
-                translated_msg = original_msg
-            translated_msgs[key] = { 'message': translated_msg }
-        else : translated_msgs[key] = messages[key]
-
-    # Format messages
-    formatted_msgs = '{\n'
-    for index, (key, message_data) in enumerate(translated_msgs.items()):
-        formatted_msg = json.dumps(message_data, ensure_ascii=False) \
-                            .replace('{', '{ ').replace('}', ' }') # add spacing
-        formatted_msgs += ( f'  "{key}": {formatted_msg}'
-                        + ( ',\n' if index < len(translated_msgs) - 1 else '\n' )) # terminate line
-    formatted_msgs += '}'
-    with open(msgs_path, 'w', encoding='utf-8') as output_file : output_file.write(formatted_msgs + '\n')
-
-    # Print file summary
-    if translated_msgs == messages : langs_skipped.append(lang_code) ; lang_skipped = True
-    elif translated_msgs != messages : langs_translated.append(lang_code) ; lang_translated = True
-    if not lang_translated : langs_not_translated.append(lang_code)
-    overwrite_print(f"{ 'Added' if lang_added else 'Skipped' if lang_skipped else 'Updated' } { folder }/messages.json")
-
-# Print final summary
-print_trunc('\nAll messages.json files updated successfully!\n')
-lang_data = [langs_translated, langs_skipped, langs_added, langs_not_translated]
-for data in lang_data:
-    if data:
-        list_name = next(name for name, value in globals().items() if value is data)
-        status = list_name.split('langs_')[-1].replace('_', ' ')
-        print(f'Languages {status}: {len(data)}\n')  # print tally
-        print('[ ' + ', '.join(data) + ' ]\n')  # list languages
+                translator = Translator(to_lang=lang_code)
+                translated_msg = translator.translate(value['message'])
+                translated_msgs[key] = {'message': translated_msg}
+            except Exception:
+                translated_msgs[key] = value
+
+    with open(msgs_path, 'w', encoding='utf-8') as output_file:
+        json.dump(translated_msgs, output_file, ensure_ascii=False, indent=4)
+
+    langs_translated.append(lang_code)
+
+print_trunc("\nTranslation process completed!\n")
+print(f"Languages translated: {len(langs_translated)}")
```

Comment:
The script won't work anymore cuz you deleted a lot of important stuff

Reply:
I will try to update the script so that the syntax is good and the important information is preserved even though the script is more "light" and efficient. I will keep you informed...

Reply:
@hexakleo please go to https://github.com/adamlui/python-utils to create each change in separate PRs